lib/Intlc/Linter.hs

```diff
@@ -67,3 +67,10 @@ lintWithRules rules (Message stream) = toStatus $ rules `flap` stream
 
 lint :: Message -> Status
 lint = lintWithRules [interpolationsRule,noAsciiRule]
+
+printLintingError :: LintingError -> IO ()
+printLintingError TooManyInterpolations           = putStrLn "Nested functions are not allowed"
+printLintingError (InvalidNonAsciiCharacter chars) = putStr "Following characters are not allowed: " >> printChars chars
+  where
+    printChars:: NonEmpty Char -> IO()
+    printChars = putStrLn . intercalate "   " . toList . fmap return
```

Comment:
I'd suggest returning `Text` inside lib so impurity is encapsulated, but otherwise I like this commit. It's awesome to see the intuition for effects building with `>>`!

FYI monadic `>>` and `return` can be replaced with their applicative cousins `*>` and `pure` respectively.